packages/fxa-js-client/tests/mocks/request.js

```diff
@@ -223,6 +227,25 @@ define(['client/lib/errors', 'tests/lib/push-constants'], function(
       status: 200,
       body: '{"subscriptions":[{"foo":"bar"}]}',
     },
+    securityEvents: {
+      status: 200,
+      body: JSON.stringify([
+        {
+          name: 'account.create',
+          verified: true,
+          createdAt: new Date().getTime(),
+        },
+        {
+          name: 'account.login',
+          verified: true,
+          createdAt: new Date().getTime(),
+        },
+      ]),
+    },
+    securityEventsEmptyResponse: {
```

Comment:
@farhan787 Sorry I meant `securityEventsEmptyResponse` should return `[]`.

Reply:
@vbudhram I was also thinking that `securityEventsEmptyResponse` should return [].